src/backend/kvm/mod.rs

```diff
@@ -1,20 +1,22 @@
 // SPDX-License-Identifier: Apache-2.0
 
+mod builder;
 mod shim;
 mod vm;
 
+pub const SHIM: &[u8] = include_bytes!(concat!(env!("OUT_DIR"), "/bin/shim-sev"));
```

Comment:
I might be missing it. But I don't think `SHIM` is actually used anywhere else. Does it need to be public?

Reply:
I would guess, not yet and will be used in a follow-up PR

Reply:
Correct ^ -- I can change it for these patches though